static/styles/zulip.css

```diff
@@ -1140,6 +1140,20 @@ a.message_label_clickable:hover {
     opacity: 1.0;
 }
 
+.on_hover_topic_mute {
+    opacity: .1;
+}
+
+.always_visible_topic_mute {
+    opacity: .7;
+}
+
+.on_hover_topic_mute:hover,
+.always_visible_topic_mute:hover {
+    cursor: pointer;
+    opacity: 1.0;
+}
+
```

Comment:
`always_visible_topic_mute` is not required. I copied the styling for the mute button from the pencil button, which used the `always_visible` class. I thought I might have to add something in `message_store.js` but apparently not.

I will correct this.